edunext_openedx_extensions/ednx_microsites/backends/filebased.py

```diff
@@ -6,32 +6,12 @@
 
 from django.conf import settings
 
-from microsite_configuration.backends.base import (
-    BaseMicrositeBackend,
-    BaseMicrositeTemplateBackend,
-)
+from .base import BaseMicrositeTemplateBackend
 from microsite_configuration.microsite import get_value as microsite_get_value
 from microsite_configuration.microsite import is_request_in_microsite
 
 
-class FilebasedMicrositeBackend(BaseMicrositeBackend):
-    """
-    Microsite backend that reads the microsites definitions
-    from a dictionary called MICROSITE_CONFIGURATION in the settings file.
-    """
-
-    def __init__(self, **kwargs):
-        super(FilebasedMicrositeBackend, self).__init__(**kwargs)
-
-
-class FilebasedMicrositeTemplateBackend(BaseMicrositeTemplateBackend):
-    """
-    Microsite backend that loads templates from filesystem.
-    """
-    pass
-
-
-class EdunextCompatibleFilebasedMicrositeTemplateBackend(FilebasedMicrositeTemplateBackend):
```

Comment:
This class was inheriting from a class that wasn't implementing any method, so better inherit from base directly